src/calculation/Calculator.ts

```diff
@@ -41,6 +41,7 @@ import { clearElmInfoCache } from '../helpers/elm/ELMInfoCache';
 import _, { omit } from 'lodash';
 import { ELM } from '../types/ELMTypes';
 import { getReportBuilder } from '../helpers/ReportBuilderFactory';
+import { option } from 'commander';
```

Comment:
Looks like an accidental import

Reply:
blargh it is!

Reply:
Resolved in [f132302](https://github.com/projecttacoma/fqm-execution/pull/299/commits/f1323025f5f095fbc6f26127fe85d26b19b7ee6d)